sound/soc/bcm/merus-amp.c

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
What does this dedicated driver offer over the support in rpi-simple-soundcard.c, other than commented out code and poor indentation?